README.md

```diff
@@ -216,5 +215,12 @@ Access tasks on this object as properties in the same way as the `obt` object.
 ## Analytics
 OBT gathers analytics data about its usage to help prioritise future development. If you would like to opt-out of analytics reporting you can set the environment variable `DISABLE_OBT_ANALYTICS_REPORTNG` to `1`.
 
+## Migration guide
+
+### Migrating from 5.X.X to 6.X.X
+
+The 6.0.0 release removes the ability for OBT to include the Babel/Core-JS polyfills from being included in the built Javascript. If you were relying on this feature, it is recommended that you use the [Polyfill service](https://polyfill.io).
```

Comment:
I think you have a double "include" here which makes this difficult to read...

Perhaps:
The 6.0.0 release removes the ability for OBT to include the Babel/Core-JS polyfills ~~from being included~~ in the built Javascript. If you were relying on this feature, it is recommended that you use the [Polyfill service](https://polyfill.io).

This is also written in the passive voice, our [documentation styleguide](https://github.com/Financial-Times/ft-origami#documentation-style-guide) covers this, but generally the active voice is better for documentation ie:

The 6.0.0 release removes the ability for OBT to include the Babel/Core-JS polyfills in the built Javascript. If you were relying on this feature, we recommend that you use the [Polyfill service](https://polyfill.io) instead.